src/lib/Form/Type/ChoiceList/Loader/TimePeriodChoiceLoader.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformAdminUi\Form\Type\ChoiceList\Loader;
+
+use Symfony\Contracts\Translation\TranslatorInterface;
+
+class TimePeriodChoiceLoader extends BaseChoiceLoader
```

Comment:
Could we rename those to 'datePeriod'? We are using 'Time' when we are have hours and minutes in mind.

Reply:
Also, we could reuse that class in basic search type instead of those private methods there :+1:

Reply:
Yes, that's the point :)

Reply:
> > Could we rename those to 'datePeriod'? We are using 'Time' when we are have hours and minutes in mind.

> > Also, we could reuse that class in basic search type instead of those private methods there  (not now, in the future)

> Yes, that's the point :)

So what's the conclusion here? ;) I'd like to see here something saying it's about dates. Though DateInterval sounds better to me.